apidef/oas/middleware.go

```diff
@@ -521,3 +524,55 @@ func (mt *MethodTransform) ExtractTo(meta *apidef.MethodTransformMeta) {
 	meta.Disabled = !mt.Enabled
 	meta.ToMethod = mt.ToMethod
 }
+
+func (ps Paths) fillCache(cacheMetas []apidef.CacheMeta) {
+	for _, cm := range cacheMetas {
+
+		plugins := ps[cm.Path].getMethod(cm.Method)
+		if plugins.Cache == nil {
+			plugins.Cache = &CachePlugin{}
+		}
+
+		plugins.Cache.Fill(cm)
+		if ShouldOmit(plugins.Cache) {
+			plugins.Cache = nil
+		}
+
+	}
+
+}
+
+func (p *Plugins) extractCacheTo(ep *apidef.ExtendedPathsSet, path string, method string) {
+
+	if p.Cache == nil {
+		return
+	}
+
+	newCacheMeta := apidef.CacheMeta{
+		CacheOnlyResponseCodes: p.Cache.CacheResponseCodes,
```

Comment:
ExtractTo function already fills this fields. Just method and path should be defined outside.